app/models/proposal.rb

```diff
@@ -49,6 +50,47 @@ class Proposal < ApplicationRecord
     declined: 9
   }
 
+  # rubocop:disable Metrics/BlockLength
```

Comment:
Please do not add rubocop skip comments - I will skip the issue in Codacy when it makes sense